### ISSUE TITLE
No Readme

### ISSUE THREAD

**Issue by lakshyabatman** [dsc-jiit-128/Todo#1]
This Repo lacks readme.
### How to solve

- Find a proper format for readme from prev repositories.
- Add details about the project
- Add details about the event.

**Comment by hoehooiyan** [comment]
Hi, may I know what is your vision about this project?

Could you give some guides on how to write the details for this project?

I wish to contribute to this todo app.

Thank you! :)

**Comment by lakshyabatman** [comment]
Yes, sure @hoehooiyan . Please refer old repos readme, this readme should include DSC logo, and some screenshots, how to use this application, and top contributor name **that's me **.

**Comment by hoehooiyan** [comment]
#3 Hi @lakshyabatman , I hava made a pull request for adding readme, please have a look.
Any changes to be made, please let me know. 
Thank you! :smile:
